timely/src/dataflow/operators/generic/builder_rc.rs

```diff
@@ -108,7 +108,7 @@ impl<G: Scope> OperatorBuilder<G> {
     /// antichain indicating that there is no connection from the input to the output.
     pub fn new_output_connection<CB: ContainerBuilder>(
         &mut self,
-        connection: Vec<Antichain<<G::Timestamp as Timestamp>::Summary>>
+        connection: PortConnectivity<<G::Timestamp as Timestamp>::Summary>
```

Comment:
This could be an iterator over `(input, summary)`, with the caveat that duplicate `input` elements will override previous mentions.

Reply:
Yeah, I thought of this, and .. figured I'd wait before being too clever. Mostly wanted the highest confidence diff first. But you are right, and it's probably tasteful to do (especially given that the `new_output` method conses up one just to have it be dropped).

Reply:
Ah, punting for the moment to think a bit more. The argument is used twice, once passed to the raw builder, which has a similar method that could have a similar change. I'd like to get my head around why everything is duplicated first.